tools/plugin-base/src/main/java/io/spine/tools/code/ExternalModule.java

```diff
@@ -18,9 +18,10 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.js.generate.resolve;
+package io.spine.tools.code;
```

Comment:
According to the class-level comment, this is a `JavaScript module`. However, as you move it out from the JS-specific package, it no longer looks as such. `NPM` reference is also misleading to me.